extensions/amp-carousel/validator-amp-carousel.protoascii

```diff
@@ -68,3 +67,61 @@ tags: {  # <amp-carousel>
     supported_layouts: RESPONSIVE
   }
 }
+tags: {  # <amp-carousel> lightbox
+  tag_name: "AMP-CAROUSEL"
+  spec_name: "amp-carousel with lightbox attribute"
```

Comment:
We typically use `spec_name` in the format `{TAG}[{attribute}]`. So this would be `AMP-CAROUSEL[lightbox]`